src/ViewUser.js

```diff
@@ -690,7 +690,116 @@ class ViewUser extends React.Component {
         onClickViewAccountActionsHistory={this.onClickViewAccountActionsHistory}
         onClickViewClosedAccounts={this.onClickViewClosedAccounts}
         onClickViewAllAccounts={this.onClickViewAllAccounts}
-      />);
+      />
+    );
+
+    if (this.isLayerOpen('add-block') || this.isLayerOpen('edit-block')) {
+      return (
+        <Layer label={query.layer === 'add-block' ? 'Add Block' : 'Edit Block'}>
```

Comment:
@DGB-UNAM-FOLIO could you please translate "Add Block" and "Edit Block"?

Reply:
Also, I believe the field is `contentLabel` which must be a string, so you probably need to wrap the whole `<Layer>` in a `<FormattedMessage>` with a render prop: ```
<FormattedMessage id="ui-users.blocks.addBlock">
  {contentLabel => (
    <Layer contentLabel={contentLabel} ...> ... </Layer>
  )}
</FormattedMessage>
```